core/domain/improvements_domain_test.py

```diff
@@ -98,22 +97,23 @@ def test_to_dict_with_non_existing_resolver_id_raises_exception(
     ) -> None:
         invalid_resolver_id = 'non_existing_user_id'
         task_entry = improvements_domain.TaskEntry(
-            improvements_models.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
-            improvements_models.TASK_TYPE_HIGH_BOUNCE_RATE,
-            improvements_models.TASK_TARGET_TYPE_STATE,
+            constants.TASK_ENTITY_TYPE_EXPLORATION, self.exp_id, 1,
+            constants.TASK_TYPE_HIGH_BOUNCE_RATE,
+            constants.TASK_TARGET_TYPE_STATE,
             feconf.DEFAULT_INIT_STATE_NAME, 'issue description',
-            improvements_models.TASK_STATUS_RESOLVED, invalid_resolver_id,
+            constants.TASK_STATUS_RESOLVED, invalid_resolver_id,
             self.MOCK_DATE)
         with self.assertRaisesRegex(Exception, 'User not found'): # type: ignore[no-untyped-call]
-            task_entry.to_dict()
+            improvements.get_task_dict_with_username_and_profile_picture( # type: ignore[no-untyped-call]
+             task_entry)
```

Comment:
Ditto as above.

Reply:
@vojtechjelinek The test here checks whether `to_dict()` raises exception for non-valid resolver ID. Since that check is now moved to `get_task_dict_with_username_and_profile_picture` (invalid resolver ID check), shouldn't htis be called here?

Reply:
But that error is no longer generated in `to_dict`, right?

Reply:
@vojtechjelinek Yeah correct. So this test should be moved to `improvements_test.py` then?

Reply:
yes, move it

Reply:
Done now.